js/modules/k6/crypto/crypto.go

```diff
@@ -168,6 +168,9 @@ func (hasher *Hasher) Digest(outputEncoding string) string {
 	case "hex":
 		return hex.EncodeToString(sum)
 
+	case "binary", "":
```

Comment:
Why is binary the default? I'm not sure this should be the case, since the current support for `binary` in JS is basically an array of ints and pretty unusable. And not a proper [`Uint8Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array), rather a simple array of ints, which is pretty terrible :slightly_frowning_face:. Currently those "binary" results are mostly useful as arguments to other k6 functions...